pkg/cloud/aws/tags/tags.go

```diff
@@ -81,19 +82,19 @@ type BuildParams struct {
 
 	// Any additional tags to be added to the resource.
 	// +optional
-	Additional Map
+	Additional v1alpha1.Map
 }
 
 // Build builds tags including the cluster tag and returns them in map form.
-func Build(params BuildParams) Map {
-	tags := make(Map)
+func Build(params BuildParams) v1alpha1.Map {
```

Comment:
I think this could safely move across to the types package too

Reply:
That's fine with me, `BuildParams` can safely be moved to v1alpha1